encord/client.py

```diff
@@ -820,6 +820,9 @@ def get_label_logs(
         self, user_hash: str = None, data_hash: str = None, from_unix_seconds: int = None, to_unix_seconds: int = None
     ) -> List[LabelLog]:
 
+        # Flag for backwards compatibility
+        include_user_email_and_interface_key = True
```

Comment:
Moving this out of the 'magic' code path and doing `query_payload['include_user_email_and_interface_key'] = True` would have saved me 10 minutes of staring at the code in an attempt to understand what's going on ;)

But it's kind of fine.